src/java.base/windows/native/libjava/gdefs_md.h

```diff
@@ -28,8 +28,3 @@
  */
 
 #include <stddef.h>
-#ifndef _WIN64
```

Comment:
I suspect the unix/windows gdefs_md.h files could be eliminated, and just make gdefs.h use portable
headers.  That can be done as a separate cleanup.

Reply:
Good point. I created https://bugs.openjdk.org/browse/JDK-8343291.